### PR TITLE
feat: Ignore by title

### PR DESCRIPTION
**Pull request by Anakael**
Implement: https://github.com/Alexays/Waybar/issues/1293

### CODE REVIEW

**Comment by Anakael** [comment]
@Alexays 
Can you review it?

File: include/modules/wlr/taskbar.hpp [hunk 1]
```diff
@@ -77,6 +77,7 @@ class Task
    private:
     std::string repr() const;
     std::string state_string(bool = false) const;
+	void hide_if_ignored();
```
Comment (Alexays) [inline include/modules/wlr/taskbar.hpp:80]: indent

File: src/modules/wlr/taskbar.cpp [hunk 2]
```diff
@@ -377,13 +376,12 @@ std::string Task::state_string(bool shortened) const
 void Task::handle_title(const char *title)
 {
     title_ = title;
+	hide_if_ignored();
```
Comment (Alexays) [inline src/modules/wlr/taskbar.cpp:379]: indent

File: src/modules/wlr/taskbar.cpp [hunk 3]
```diff
@@ -397,6 +395,12 @@ void Task::handle_app_id(const char *app_id)
           handle_output_enter(output);
         }
     }
+}
+
+void Task::handle_app_id(const char *app_id)
+{
+    app_id_ = app_id;
+	hide_if_ignored();
```
Comment (Alexays) [inline src/modules/wlr/taskbar.cpp:403]: indent

File: src/modules/wlr/taskbar.cpp [hunk 4]
```diff
@@ -276,8 +276,7 @@ Task::Task(const waybar::Bar &bar, const Json::Value &config, Taskbar *tbar,
         struct zwlr_foreign_toplevel_handle_v1 *tl_handle, struct wl_seat *seat) :
     bar_{bar}, config_{config}, tbar_{tbar}, handle_{tl_handle}, seat_{seat},
     id_{global_id++},
-    content_{bar.vertical ? Gtk::ORIENTATION_VERTICAL : Gtk::ORIENTATION_HORIZONTAL, 0},
-    button_visible_{false}, ignored_{false}
```
Comment (Alexays) [inline src/modules/wlr/taskbar.cpp:280]: normal that it is removed?

Comment (Anakael) [reply src/modules/wlr/taskbar.cpp:280]: Yes. Because `button_visible_` and `ignored_` are initialized with default value in taskbar.hpp

**Comment by Alexays** [comment]
LGTM!

**Pull request merged**